corehq/apps/users/management/commands/gdpr_scrub_user_from_forms.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from io import StringIO
+from lxml import etree
+import sys
+import six
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Scrubs the username from all forms associated with the given user"
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+        parser.add_argument('domain')
+
+    def handle(self, username, domain, **options):
+        this_form_accessor = FormAccessors(domain=domain)
+        user = CommCareUser.get_by_username(username)
```

Comment:
This could be a web user too. Probably better to use `CouchUser.get_by_username`

Reply:
Fixed here (and learned the differences between types of users +1)
https://github.com/dimagi/commcare-hq/pull/19979/commits/de2e98ad22ae74f559608bbb2f0a2194a7599cdf